contrib/local-mainnet/zetacored/configs/app.toml

```diff
@@ -0,0 +1,77 @@
+minimum-gas-prices = "1.0azeta"
```

Comment:
This should be 10000000000azeta. 
gas prices on all networks were increased to 10Gwei/10000000000azeta